catalog/spatial/registry/registry-source-configuration-handler/src/main/java/org/codice/ddf/registry/source/configuration/SourceConfigurationHandler.java

```diff
@@ -416,13 +419,14 @@ private void deleteRegistryConfigurations(Metacard metacard)
    */
   private Configuration toggleConfiguration(Configuration config) throws IOException {
     String newFpid;
+    Dictionary<String, Object> properties;
     if (config.getFactoryPid().contains(DISABLED_CONFIGURATION_SUFFIX)) {
       newFpid = config.getFactoryPid().replace(DISABLED_CONFIGURATION_SUFFIX, "");
-
+      properties = copyConfigProperties(config.getProperties(), newFpid);
     } else {
       newFpid = config.getFactoryPid().concat(DISABLED_CONFIGURATION_SUFFIX);
```

Comment:
✏️ Extract the factory pid in a local variable instead of retrieving twice from `config`